src/scripts/main.js

```diff
@@ -1,7 +1,37 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const container = document.createElement('container');
```

Comment:
The element 'container' is not a valid HTML tag. You should use 'div' or another valid HTML element instead.